changelog.d/9150.feature

```diff
@@ -0,0 +1,4 @@
+New API /_synapse/admin/rooms/{roomId}/context/{eventId}
+
+This API mirrors /_matrix/client/r0/rooms/{roomId}/context/{eventId} but lets administrators
+inspect rooms. Designed to annotate abuse reports with context.
```

Comment:
```suggestion
Add a new admin API for getting event contexts.
```

We generally keep changelogs as one line, if people are interested in more info the changelog includes a link to the PR :)